platform-coop-toolkit/pinecone
# feat!: add comprehensive Django support to inputs

## Description

Adds support for alternative markup for all form inputs (based on Django conventions).

- [x] Generic inputs (`text`, `password`, `email`, `url`, `tel`, `number`)
- [x] Text areas
- [x] Selects
- [x] Checkboxes
- [x] Custom checkboxes (with indeterminate state)
- [x] Radio buttons 

## Steps to test

1. Visit deploy preview.
2. Test all form elements, as well as filter UI and dialog with input.

**Expected behavior:** Everything works and looks as expected.

## Additional information

Nested checkboxes now require an `.input--parent` class on the parent checkbox. See: https://github.com/platform-coop-toolkit/pinecone/pull/376/files#diff-03a19b9935aa0f00880bc05b3fc94effR47

## Related issues

- Resolves #369.
- Resolves #370.
